Translated/Dialog/Deirdre/ssn_014270.csv

```diff
@@ -0,0 +1,12 @@
+name_7400#0,"""Deirdre"""
+ssn_014270_0010#0,"""Hey, ain't seen you around here before. Newly arrived to Central City, are ya?"""
+ssn_014270_0020#0,"""You should come see me after you get some notches on your belt. Might be as I can teach you a thing or two."""
+ssn_014270_0030#0,"""Good seein' you."""
+ssn_014270_0040#0,"""I'm the head instructor for hybrid combat."""
+ssn_014270_0050#0,"""I've heard about you. Making quite the name for yourself lately. I like that."""
+ssn_014270_0055#0,"""Wanna learn about hybrid combat? I can teach you."""
+ssn_014270_0060#0,"""Oh well... Let me know if you change your mind."""
+ssn_014270_0070#0,"""No special treatment here, you start from the basics like everyone else."""
+ssn_014270_0080#0,"""By the time I'm done with you, you'll fight as naturally as you breathe."""
+ssn_014270_0101#0,"""You should have learned about the importance of Class Skills from your basic training."""
+ssn_014270_0102#0,"""Bravers are the architects of combat. When you've reached the peak, just keep going. Dare to climb even higher."""
```

Comment:
Should this be something like "pioneers", not architects?

Reply:
decided to just change the translation to actually reflect the japanese lmao